apps/hash-api/src/graphql/resolvers/knowledge/entity/entity.ts

```diff
@@ -130,7 +134,7 @@ export const queryEntitiesResolver: Extract<
   const { data: entitySubgraph } = await graphApi.getEntitiesByQuery({
     filter,
     graphResolveDepths: {
-      inheritsFrom: { outgoing: 0 },
+      ...zeroedGraphResolveDepths,
```

Comment:
Upon seeing this diff I'm not _sure_ I like this pattern. If we added a new depth to the query, or originally had forgotten to pass one of these (`constrainsValuesOn` for example) then there wouldn't be an error here.

Also, looking at the code, it's now unclear what _is_ zeroed, we have to mentally apply the difference between the full object and the relatively long list of things here..

Maybe the code ergonomics are worth it, but I'm not sure I lean that way right now

Reply:
Do you suggest to remove it from this code section, remove the constant everywhere, or just leave it in? It feels like `Partial<GraphResolveDepths>` generally is an issue.

Reply:
I'm not sure I'm proposing anything, we can leave it as is, just wanted to express some thoughts